sceptre-example/templates/custom-image/dockerfile

```diff
@@ -0,0 +1,2 @@
+# You can build and push custom dockerfiles to ECR. This is just a demonstration.
+FROM python:3.9
```

Comment:
Is there supposed to be more to this? Or will this actually be enough to push to ECR?

Reply:
This is actually enough, believe it or not. For purposes of demonstration, it's all we really need.